docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -95,27 +104,66 @@ Before using this method, you must have set up a https://learn.microsoft.com/en-
 . Go to your Azure subscription list and select the subscription you want to monitor with CSPM.
```

Comment:
What if you want to monitor an organization instead of a single subscription? Should this step be different?